src/components/Modal/Modal.mdx

```diff
@@ -124,7 +125,25 @@ import { ModalTrigger } from "@nypl/design-system-react-components";
   modalProps={{
     bodyContent: "body text",
     closeButtonLabel: "Close Button",
-    headingText: "Modal Heading Text",
+    headingText: (
+      <Heading
+        level="h3"
+        display="flex"
+        alignItems="center"
+        gap="xs"
+        size="heading5"
+      >
+        <>
+          <Icon
+            align="left"
```

Comment:
Remove this.  All it does is add extra margin on the right side and it throws off the spacing.  I never liked this prop.